### DISCUSSION TITLE
Play local File on HASSIO

### DISCUSSION THREAD

**Discussion by Alex-joomla** [hklages/node-red-contrib-sonos-plus#173] [Q&A] [answered]
Can you tell my what will be the quickest way to play a local file from home assistant on my sonos device?

Tried a function node with
`msg.payload = "play.notification";
msg.volume = "40";
msg.topic = "http://192.168.0.15/local/media/doorbell_ding-dong.mp3";
return msg;`

but it didn't work.

**Comment by hklages** [comment]
Assuming, that media/doorbell_ding-dong.mp3 is under directory www, then only the port 8123 is missing. 

`http://192.168.0.15:8123/local/media/doorbell_ding-dong.mp3`

**Comment by Alex-joomla** [comment]
Thank you for the hint with the port! I still get this error:

`"processInputMsg: command is invalid >>group.http://192.168.0.15:8123/local/media/doorbell_ding-dong.mp3  :: Details: none"`

**Comment by hklages** [comment, marked as answer]
`group.http:// `.... is for sure an invalid command.
use command: `group.play.notification`
payload: `http://192.168.0.15:8123/local/media/doorbell_ding-dong.mp3`
